src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb

```diff
@@ -0,0 +1,442 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "disturbed-butterfly",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from software.proto.messages_robocup_ssl_wrapper_pb2 import SSL_WrapperPacket\n",
+    "from python_tools.proto_log import ProtoLog\n",
+    "\n",
+    "wrapper_proto_log = ProtoLog(\"/home/jordan/log_ssl_wrapper_filtered_world_test/SensorFusion_SSL_WrapperPacket\", \n",
```

Comment:
I obviously don't have this directory on my computer. Can the ProtoLog function fail fast when the directory doesn't exist?

Reply:
Opened #1909